src/intl/en.json

```diff
@@ -233,11 +233,11 @@
   "page-eth-flexible-amounts": "Available in flexible amounts",
   "page-eth-flexible-amounts-desc": "ETH is divisible up to 18 decimal places so you don't have to buy 1 whole ETH. You can buy fractions at a time – as little as 0.000000000000000001 ETH if you want.",
   "page-eth-monetary-policy": "Ethereum's monetary policy",
-  "page-eth-ethhub": "EthHub",
+  "page-eth-ethhub-caption": "Updated often",
   "page-eth-value": "Why Ether is valuable",
-  "page-eth-sassano": "Anthony Sassano",
+  "page-eth-last-updated": "January 2019",
```

Comment:
We shouldn't require our volunteers to translate dates like this. Check out our [time utils](https://github.com/ethereum/ethereum-org-website/blob/dev/src/utils/time.js) & the library we use: https://moment.github.io/luxon/

Using that library, we should be able to pass strings like "January 2019" to a function that translates & formats them depending on the user's current language.